### DISCUSSION TITLE
Ability to generate a trial balance with drafts or batch to be approved

### DISCUSSION THREAD

**Discussion by ehuelsmann** [ledgersmb/LedgerSMB#6969] [Ideas]
In order to be able to quickly assess whether to approve a batch or draft,it would be extremelypractical to be able to run a trail balance with the mutation columns composed of the line items in the batch/draft.

**Comment by einhverfr** [comment]
Also will add to gl search

**Comment by einhverfr** [comment]
hmm misread this. adding the ability to include these on the trial balance though so that people with longer approval times are happy :-)
